### PR TITLE
CTCP-5925: Review test coverage percentage

### PR DESCRIPTION
**Pull request by justega247**

### CODE REVIEW

**Review by kevincoatsworth — DISMISSED** [review]

File: test/generators/ModelGenerators.scala [hunk 1]
```diff
@@ -129,6 +129,17 @@ trait ModelGenerators {
       Gen.oneOf(Locked, Unlocked, LockCheckFailure)
     }
 
+  lazy val submissionState: Arbitrary[SubmissionState] = Arbitrary {
```
Comment (jackheslop96) [inline test/generators/ModelGenerators.scala:132]: Could make this one `implicit` as this truly is an arbitrary submission state, also consider naming it `arbitrarySubmissionState`

File: test/models/SubmissionStateSpec.scala [hunk 2]
```diff
@@ -30,14 +30,14 @@ class SubmissionStateSpec extends AnyFreeSpec with Generators with Matchers with
   "submissionState" - {
 
     "must deserialise" in {
-      forAll(arbitrary[SubmissionState]) {
+      forAll(arbitrary[SubmissionState](submissionState)) {
```
Comment (jackheslop96) [inline test/models/SubmissionStateSpec.scala:33]: This change could then be reverted

File: test/models/SubmissionStateSpec.scala [hunk 3]
```diff
@@ -30,14 +30,14 @@ class SubmissionStateSpec extends AnyFreeSpec with Generators with Matchers with
...
         state =>
           JsString(state.asString).as[SubmissionState] mustEqual state
       }
     }
 
     "must serialise" in {
-      forAll(arbitrary[SubmissionState]) {
+      forAll(arbitrary[SubmissionState](submissionState)) {
```
Comment (jackheslop96) [inline test/models/SubmissionStateSpec.scala:40]: This change could then be reverted

File: test/handlers/ErrorHandlerSpec.scala [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    override def method: String = "POST"
+
+    override def version: String = "HTTP/1.1"
+
+    override def headers: Headers = new Headers(Seq.empty)
+
+    override def connection: RemoteConnection = RemoteConnection("", secure = true, None)
+
+    override def attrs: TypedMap = TypedMap()
+  }
```
Comment (jackheslop96) [inline test/handlers/ErrorHandlerSpec.scala:105]: This might be redundant. Does `fakeRequest` in SpecBase work as a valid RequestHeader?

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**